src/locales/en.json

```diff
@@ -297,7 +297,7 @@
   "errors": {
     "invalid_capture": {
       "message":"No document detected",
-      "instruction": "Make sure all the document is in the photo"
+      "instruction": "Make sure all of the document is in the photo"
```

Comment:
I thought I had removed `en.json` and `es.json`. I don't think these files are in use anymore.

Reply:
Good to know. I'll remove them from this PR then. We might need to look out for them when merging.